docs/usage/2-manifest.md

```diff
@@ -88,6 +107,39 @@ Each target in the list of project targets can be initialized with the following
   - **CurrentVersion:** Current version without the extension.
 - **Environment (optional):** It's a `[String: String]` to defined variables that will be set to the scheme that Tuist automatically generates for the target.
 
+### Scheme
+
+A `Scheme` will help to define a collection of targets to `Build, Run, Test, Profile, Analyze and Archive`:
+> Note: Tuist will auto-generate Scheme for each target if they are not defined. 
+- **name: String:** Name of Scheme
+- **shared: Bool:** Scheme is shared or not 
+- **buildAction: BuildAction?:** Scheme Build options
+    - **targets: [String]:** A list of targets will be built, which are defined in Project
```

Comment:
```suggestion
    - **targets: [String]:** A list of targets to build, which are defined in the Project
```

Reply:
This actually surfaces a future enhancement - technically Xcode does all you to specify any target in the workspace not just the project 🙂

Reply:
Yes, at the moment we cannot generate Scheme with targets from different Project as Xcode does.